src/bin/bat/app.rs

```diff
@@ -226,17 +225,14 @@ impl App {
             style_components,
             syntax_mapping,
             pager: self.matches.value_of("pager"),
-            use_italic_text: match self.matches.value_of("italic-text") {
-                Some("always") => true,
-                _ => false,
-            },
+            use_italic_text: matches!(self.matches.value_of("italic-text"), Some("always")),
```

Comment:
Couldn't we simply use
```
self.matches.value_of("italic-text") == Some("always")
```
?

We're not really using the powerful feature of the `matches!` macro here.